src/app/history/history-bet/history-bet.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="history-container">
+  <div class="history-title"><h3>History of your bets</h3></div>
+  <div class="history-elements">
+    <app-history-bet-item class="item-background"></app-history-bet-item>
```

Comment:
why one item is a separate component? I would suggest to keep whole table as one component and do not decompose by rows as components

Reply:
done, changed